ow_lander/srv/GuardedMove.srv

```diff
@@ -1,13 +1,11 @@
 bool use_defaults           # Set to true to use the default touch location
-bool delete_prev_traj       # Set to true to cleanup ~/.ros
```

Comment:
If you remove delete_prev_traj from the .srv file, you should remove it from activity_guarded_move.py>arg_parsing , otherwise it will look for the value of this variable, which doesn't exist, and the service doesn't run. Same is valid for deliver_sample.

Reply:
addressed

Reply:
$ rosservice call /arm/guarded_move "{use_defaults: true, x: 0.0, y: 0.0, z: 0.0, direction_x: 0.0, direction_y: 0.0,
  direction_z: 0.0, search_distance: 0.0}" 
ERROR: service [/arm/guarded_move] responded with an error: error processing request: global name 'delete_prev_traj' is not defined

Is this the same delete_prev_traj problem that @damianacatanoso had? I still have it and my branches are up-to-date. I also get the same error from deliver_sample with use_defaults = true.

Reply:
I get the same error. Let me check what is the issue!

Reply:
I probably didn't test this part of the code as I should've, so really thanks! I have pushed needed changes to address the problem!